packetbeat/protos/http/event.go

```diff
@@ -36,6 +36,9 @@ type ProtocolFields struct {
 	// "Lowercase Capitalization" in the "Implementing ECS"  section.
 	RequestMethod common.NetString `ecs:"request.method"`
 
+	// HTTP request ID.
+	RequestID common.NetString `ecs:"request.id"`
```

Comment:
This one was intended right?

Reply:
Yes, it's required for unit test to pass :)